tests/test_fixtures.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+import pytest
+
+from tests.fixtures.snapshot_date import validate_time
+
+
+def test_time_invalid():
+    timestamp = "2020/09/26"
+    with pytest.raises(ValueError):
+        validate_time(timestamp)
+
+
+def test_time_recent():
+    timestamp = datetime.datetime.utcnow()
+    assert validate_time(timestamp) == "approximately_now"
+
+
+def test_time_not_recent():
+    timestamp = datetime.datetime.utcnow() - datetime.timedelta(minutes=1)
+    assert validate_time(timestamp) == "not_approximately_now"
```

Comment:
* Test the actual `snapshot_recent` fixture.
* Once you update things to support ISO formatted strings, make sure these tests cover ISO vs incorrect format and datetime vs str type.